.github/workflows/packaging.yml

```diff
@@ -0,0 +1,29 @@
+name: Package
+on:
+  pull_request_target:
+    types: [closed]
+jobs:
+  package:
+    name: Package and push
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+           persist-credentials: false
+      - uses: cloudve/helm-ci@master
+        with:
+          chart-name: cloudlaunchserver
+          charts-repo: cloudve/helm-charts
+          github-token: ${{ secrets.CHARTS_TOKEN }}
+          charts-token: ${{ secrets.CHARTS_TOKEN }}
+          github-labels: ${{ join(github.event.pull_request.labels.*.name, ', ') }}
+          git-branch: ${{ github.event.pull_request.base.ref }}
+      - uses: cloudve/helm-ci@master
+        with:
+          chart-name: cloudlaunch
+          charts-repo: cloudve/helm-charts
+          github-token: ${{ secrets.CHARTS_TOKEN }}
+          charts-token: ${{ secrets.CHARTS_TOKEN }}
+          github-labels: ${{ join(github.event.pull_request.labels.*.name, ', ') }}
+          git-branch: ${{ github.event.pull_request.base.ref }}
```

Comment:
How is the dependency handled here? Is the requirements.yml for cloudlaunch also bumped so that the latest cloudlaunchserver packaged chart is fetched when repackaging cloudlaunch?

Reply:
It's not handled right now. My thought was let's do the change to Helm 3 and merge it into Chart.yaml and then it'll be easier to handle!

Reply:
Should be possible with https://github.com/CloudVE/cloudlaunch-helm/pull/19 and https://github.com/CloudVE/helm-ci/pull/6

Reply:
Seems to work in https://github.com/CloudVE/cloudlaunch-helm/pull/18/commits/302771c2d0c1398d36a5f6e59b90cbfffe3ab107